### PR TITLE
[webview-docs] Elaborate on using local content with CORS protection. Fixes JB#57956

### PR DESCRIPTION
**Pull request by llewelld**
Adds a description to the docs of three different approaches that can be used for serving local content using a WebView, in order to avoid Cross Origin Resource Sharing protection issues.

### CODE REVIEW

**Comment by llewelld** [comment]
@poetaster: it'd be good to get your thoughts on this.

**Comment by poetaster** [comment]
> @poetaster: it'd be good to get your thoughts on this.

If just read through it and, as usual, clear and comprehensive! This will be of utility to many frustrated devs :)

File: doc/index.qdoc [hunk 1]
```diff
@@ -115,34 +116,143 @@ WebView currently supports customization via the following interfaces:
...
+
+\section3 Option 1. Don't serve the files locally at all
+
+In many cases, while it can initially seem sensible to serve local files,
+you may find you can get the same or better capabilities by serving a
+remote version of your app instead.
+
+Many Web Apps will use caching, \l {https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API}{Web Storage}
+and similar technologies so that they can operate fully even when your
+phone is offline. The app will be served from the remote server on first
```
Comment (rainemak) [inline doc/index.qdoc:155]: so that they can operate fully even when offline.

Simpler not to mention "phone".

**Review by rainemak — APPROVED** [review]
Added a minor comment. Approving already as this looks like a good improvement / clarification.

File: doc/index.qdoc (hunk 1 above)
Comment (llewelld) [reply doc/index.qdoc]: Yeah, that's better. Thank you.

**Pull request merged**